### PR TITLE
Update toggl-dev

### PR DESCRIPTION
**Pull request by yurikoles**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/toggl-dev.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'toggl-dev' do
-  version '7.4.9010'
-  sha256 '554a4dcf24c88ea8c5318072be55d592b4a3258fd15f7d59aafb4f6fb63874ee'
+  version '7.4.1065'
+  sha256 '7d5a5d62dbb97d3074ddfc29653279ee9c1b63c03579e0054093d418da41c1d8'
 
-  # github.com/toggl/toggldesktop was verified as official when first introduced to the cask
-  url "https://github.com/toggl/toggldesktop/releases/download/v#{version}/TogglDesktop-#{version.dots_to_underscores}.dmg"
+  # github.com/toggl-open-source/toggldesktop was verified as official when first introduced to the cask
+  url "https://github.com/toggl-open-source/toggldesktop/releases/download/v#{version}/TogglDesktop-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/toggl-dev.rb]: Help me get here from the `homepage`.

File: Casks/toggl-dev.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 cask 'toggl-dev' do
-  version '7.4.9010'
-  sha256 '554a4dcf24c88ea8c5318072be55d592b4a3258fd15f7d59aafb4f6fb63874ee'
+  version '7.4.1065'
```
Comment (vitorgalvao) [inline Casks/toggl-dev.rb]: Appcast still shows `7.4.1064`.

File: Casks/toggl-dev.rb (hunk 1 above)
Comment (yurikoles) [reply Casks/toggl-dev.rb]: I'm not sure, but old builds was removed, that's why I had created this patch.

Comment (yurikoles) [reply Casks/toggl-dev.rb]: For example from appcast?

Comment (yurikoles) [reply Casks/toggl-dev.rb]: Знімок екрана 2019-12-05 о 10 32 16

**Comment by yurikoles** [comment]
@vitorgalvao fixed, please merge

**Pull request merged**